[2024-09-24T16:17:26.935+0200] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-24T16:17:26.947+0200] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mi_dag_api.get_json_from_task1 scheduled__2023-10-06T00:00:00+00:00 [queued]>
[2024-09-24T16:17:26.960+0200] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mi_dag_api.get_json_from_task1 scheduled__2023-10-06T00:00:00+00:00 [queued]>
[2024-09-24T16:17:26.960+0200] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-09-24T16:17:26.986+0200] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): get_json_from_task1> on 2023-10-06 00:00:00+00:00
[2024-09-24T16:17:26.989+0200] {standard_task_runner.py:72} INFO - Started process 42641 to run task
[2024-09-24T16:17:26.993+0200] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'mi_dag_api', 'get_json_from_task1', 'scheduled__2023-10-06T00:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/dag_1.py', '--cfg-path', '/tmp/tmpftzwo_rw']
[2024-09-24T16:17:26.994+0200] {standard_task_runner.py:105} INFO - Job 9: Subtask get_json_from_task1
[2024-09-24T16:17:27.056+0200] {task_command.py:467} INFO - Running <TaskInstance: mi_dag_api.get_json_from_task1 scheduled__2023-10-06T00:00:00+00:00 [running]> on host DESKTOP-EPJQTCS.
[2024-09-24T16:17:27.196+0200] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='mi_dag_api' AIRFLOW_CTX_TASK_ID='get_json_from_task1' AIRFLOW_CTX_EXECUTION_DATE='2023-10-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-06T00:00:00+00:00'
[2024-09-24T16:17:27.197+0200] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-24T16:17:27.231+0200] {logging_mixin.py:190} INFO - {'cod': 401,
 'message': 'Invalid API key. Please see '
            'https://openweathermap.org/faq#error401 for more info.'}
[2024-09-24T16:17:27.231+0200] {python.py:240} INFO - Done. Returned value was: None
[2024-09-24T16:17:27.243+0200] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-24T16:17:27.244+0200] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=mi_dag_api, task_id=get_json_from_task1, run_id=scheduled__2023-10-06T00:00:00+00:00, execution_date=20231006T000000, start_date=20240924T141726, end_date=20240924T141727
[2024-09-24T16:17:27.288+0200] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-24T16:17:27.318+0200] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-24T16:17:27.334+0200] {local_task_job_runner.py:245} INFO - ::endgroup::
